Userland/Utilities/grep.cpp

```diff
@@ -293,6 +296,14 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         return did_match_something ? 0 : 1;
     };
 
+    if (fixed_strings) {
+        Vector<Regex<Fixed>> regular_expressions;
+        for (auto pattern : patterns) {
```

Comment:
This is both smart and weird tbh, feels like regex shouldn't even be involved...